lib/providers/snackbar/snackbarContext.tsx

```diff
@@ -0,0 +1,11 @@
+// SnackbarContext.js
+import { SnackbarProps } from "../../components/Snackbar/Snackbar";
```

Comment:
let's use alias when importing

```suggestion
import { SnackbarProps } from "@components/Snackbar/Snackbar";
```